diag_manager/fms_diag_field_object.F90

```diff
@@ -1105,33 +1105,33 @@ end subroutine get_dimnames
 
 !> @brief Wrapper for the register_field call. The select types are needed so that the code can go
 !! in the correct interface
-subroutine register_field_wrap(fileobj, varname, vartype, dimensions)
-  class(FmsNetcdfFile_t),            INTENT(INOUT) :: fileobj       !< Fms2_io fileobj to write to
-  character(len=*),                  INTENT(IN)    :: varname       !< Name of the variable
-  character(len=*),                  INTENT(IN)    :: vartype       !< The type of the variable
-  character(len=*), optional,        INTENT(IN)    :: dimensions(:) !< The dimension names of the field
+subroutine register_field_wrap(fms2io_fileobj, varname, vartype, dimensions)
+  class(FmsNetcdfFile_t),            INTENT(INOUT) :: fms2io_fileobj  !< Fms2_io fileobj to write to
+  character(len=*),                  INTENT(IN)    :: varname         !< Name of the variable
```

Comment:
I don't think you should add these spaces to line up the comments because then YOU become the owner of those lines.  I think you can remove the spaces between `fms2io_fileobj` and `!<` instead to make the comments line up.